examples/compute_static_flow_pressure.py

```diff
@@ -27,8 +27,9 @@
 
 from astrovascpy import bloodflow
 from astrovascpy.io import load_graph_from_bin
-from astrovascpy.io import load_graph_from_csv
-from astrovascpy.io import load_graph_from_h5
+
+# from astrovascpy.io import load_graph_from_csv
```

Comment:
This dead code should be removed.

Reply:
Let's keep this commented code for now.  It is not part of the library, it is just for this example. 
It is useful to switch quickly when loading different datasets during the analysis.

Reply:
I will probably suggest a PR later on. For now, that is fine.